index.src.html

```diff
@@ -108,9 +108,9 @@
       text: name; for: meta; url: attr-meta-name
       text: referrerpolicy; for: a; url: attr-a-referrerpolicy
       text: referrerpolicy; for: link; url: attr-link-referrerpolicy
-spec: MIX; urlPrefix: https://w3c.github.io/webappsec/specs/mixedcontent/
+spec: SECURE-CONTEXTS; urlPrefix: https://www.w3.org/TR/secure-contexts
```

Comment:
Wrong URL, should be https://w3c.github.io/webappsec-secure-contexts/ ("TR stands for trash")